src/packages/media/media/workspace/views/info/media-workspace-view-info-reference.element.ts

```diff
@@ -135,7 +135,7 @@ export class UmbMediaWorkspaceViewInfoReferenceElement extends UmbLitElement {
 	}
 
 	#renderItems() {
-		if (!this._items?.length) return html`<p>${this.localize.term('references_DataTypeNoReferences')}</p>`;
+		if (!this._items?.length) return html`<p>${this.localize.term('references_itemHasNoReferences')}</p>`;
```

Comment:
This is perhaps out of scope, but I would really prefer if we used umb-localize where possible because you then move the listener down to that element so it doesn't have to update all the time.

```suggestion
		if (!this._items?.length) return html`<p><umb-localize key="references_itemHasNoReferences">This item has no references.</umb-localize></p>`;
```

This also allows us to more easily be able to see what the translation is supposed to say 👍

Reply:
Good point! 👍 I'll keep in mind to fix in other places too.